### DISCUSSION TITLE
Generated changelog uses incorrect heading level

### DISCUSSION THREAD

**Discussion by trallnag** [semantic-release/semantic-release#2312] [Q&A] [answered]
I'm trying out semantic-release on a small project. It works great. Thanks for that. My problem: I noticed something weird about the generated changelog. Patch level releases use the wrong heading level (## instead of ###). I have not configured anything special. When I compare it with other repositories that use semantic-release something is wrong. But I don't know what.

Do you have any idea?

Here is the repository where I noticed the issue: <https://github.com/trallnag/ansible-role-asdf-plugin>

Permalink to .releaserc.json: <https://github.com/trallnag/ansible-role-asdf-plugin/blob/5ac19749322d0ada0a582987227b8218dc07dd48/.releaserc.json>

Permalink to package.json: <https://github.com/trallnag/ansible-role-asdf-plugin/blob/5ac19749322d0ada0a582987227b8218dc07dd48/.local/node/package.json>

Permalink to "broken" Changelog: <https://github.com/trallnag/ansible-role-asdf-plugin/blob/5ac19749322d0ada0a582987227b8218dc07dd48/CHANGELOG.md>

### How it looks like

<https://github.com/trallnag/ansible-role-asdf-plugin/blob/master/CHANGELOG.md>

![image](https://user-images.githubusercontent.com/24834206/147288856-0bd55031-e38d-4ee4-bcb3-7e3d891c67e1.png)

```
# Changelog

### [1.1.5](https://github.com/trallnag/ansible-role-asdf-plugin/compare/1.1.4...1.1.5) (2021-12-17)

### 🐛 Bug Fixes
```

### How it should look like

<https://github.com/oleg-koval/semantic-release-npm-github-publish/blob/master/CHANGELOG.md>

![image](https://user-images.githubusercontent.com/24834206/147288950-58546bb1-3716-4132-aace-64536aa26501.png)

```
## [1.5.1](https://github.com/oleg-koval/semantic-release-npm-github-publish/compare/v1.5.0...v1.5.1) (2021-09-23)

### chore
```

**Comment by travi** [comment, marked as answer]
semantic-release uses conventional-changelog for generation of the changelog. looking for similar issues there, i found https://github.com/conventional-changelog/conventional-changelog/issues/867. you may want to check for more issues there to see where they are on this topic

**Reply by trallnag** [reply]
Thanks a lot for the pointer! I only checked semantic-release/changelog and semantic-release/release-notes-generator
